src/dlgButtonSelect.cpp

```diff
@@ -0,0 +1,233 @@
+/***************************************************************************
+ *   Copyright (C) 2008-2013 by Heiko Koehn - KoehnHeiko@googlemail.com    *
+ *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
+ *   Copyright (C) 2016 by Stephen Lyons - slysven@virginmedia.com         *
+ *   Copyright (C) 2017 by Ian Adkins - ieadkins@gmail.com                 *
```

Comment:
Haha. Well, I was just trying to be safe since I copy-pasted the original code from something else with those copyrights already on it. Am I safe to remove the other copyrights if I'm still using code I shamelessly copy/pasted from another file?